erpnext/selling/report/payment_terms_status_for_sales_order/payment_terms_status_for_sales_order.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# License: MIT. See LICENSE
+
+import frappe
+from frappe import _, qb, query_builder
+from frappe.query_builder import functions
+
+
+def get_columns():
+	columns = [
+		{
+			"label": _("Sales Order"),
+			"fieldname": "name",
+			"fieldtype": "Link",
+			"options": "Sales Order",
+			"read_only": 1,
+		},
+		{
+			"label": _("Submitted"),
+			"fieldname": "submitted",
+			"fieldtype": "Date",
+			"read_only": 1
+		},
+		{
+			"label": _("Payment Term"),
+			"fieldname": "payment_term",
+			"fieldtype": "Data",
+			"read_only": 1
+		},
+		{
+			"label": _("Description"),
+			"fieldname": "description",
+			"fieldtype": "Data",
+			"read_only": 1
+		},
+		{
+			"label": _("Due Date"),
+			"fieldname": "due_date",
+			"fieldtype": "Date",
+			"read_only": 1
+		},
+		{
+			"label": _("Invoice Portion"),
+			"fieldname": "invoice_portion",
+			"fieldtype": "Percent",
+			"read_only": 1,
+		},
+		{
+			"label": _("Payment Amount"),
+			"fieldname": "payment_amount",
+			"fieldtype": "Currency",
+			"read_only": 1,
+		},
+		{
+			"label": _("Paid Amount"),
+			"fieldname": "paid_amount",
+			"fieldtype": "Currency",
```

Comment:
Different companies could have different default currencies. Currently for all companies current from Global Defaults will be picked. You'll have to add a hidden column for current and use that as Options for all the Currency columns to avoid this

Reply:
added new hidden column as suggested